system/default_ad_api/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>default_ad_api</name>
+  <version>0.0.0</version>
```

Comment:
[Nits] Use 0.1.0?
https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

```suggestion
  <version>0.1.0</version>
```